src/index.js

```diff
@@ -78,4 +78,13 @@ const resolveParams = args => {
   return cachePlugin(args)
 }
 
+// expose plugin as default
 export default resolveParams
+
+// expose action enhancer
+export function cacheAction(action) {
+  return function cacheEnhancedAction(context, payload) {
+    cachePlugin(context)
```

Comment:
Well @superwf was faster than I could address few changes to the code in order not to break control over cache. This would take place in another PR. Thanks for your feedback @VitorLuizC